### ISSUE TITLE
elements bundle not loading in IE 10/11

### ISSUE THREAD

**Issue by ciampo** [ebidel/polymer-gmail#10]
I'm working to a Polymer app, and I'm trying to make the page load fast by:
- loading app.js and elements.html asynchronously (with the ```async``` tag set in ```index.html```)
- loading the WebComponents polyfill only if necessary

I had to give up on the async load of ```elements.html```, because of Polymer's buggy behaviour (see [issue 248](https://github.com/Polymer/polymer/issues/248) and [issue 2522](https://github.com/Polymer/polymer/issues/2522)).

But even after that, the elements.html file won't load in Internet Explorer 10/11. I had a look to this polymer-gmail project after I saw the talk at the Polymer Summit 2015, and I found the same problem: the ```elements.html``` file just doesn't get downloaded in IE10 / 11.

Any idea?

**Comment by ebidel** [comment]
I need more info. Are you able to get it working if wc.js is always loaded, and not async? It could be a timing issue with the polyfills. What does the network panel show?

**Comment by ciampo** [comment]
Yes, yesterday I tried to always load the WC polyfill in a render-blocking way, and it seemed to fix the problem.

I'll take a deeper look and I'll come back with more info
